src/kirby/components/alert/alert.component.ts

```diff
@@ -0,0 +1,76 @@
+import { NavParams } from '@ionic/angular';
+
+import { Component, ElementRef, ViewChild, AfterViewInit, OnInit } from '@angular/core';
+
+import { AlertConfig } from '../modal/alert/config/alert-config';
+import { IModalController } from '../modal/services/modal.controller.interface';
+import { ModalHelper } from '@kirbydesign/designsystem/helpers/modal-helper';
+
+@Component({
+  templateUrl: './alert.component.html',
+  styleUrls: ['./alert.component.scss'],
+})
+export class AlertComponent implements AfterViewInit, OnInit {
+  @ViewChild('alertWrapper') alertWrapper: ElementRef;
+  config: AlertConfig;
+  scrollY: number = Math.abs(window.scrollY);
+  okBtn: string;
+  okBtnIsDestructive: boolean;
+  cancelBtn: string;
+  title: string;
+  message: string;
+  iconName: string;
+  iconThemeColor: string;
+
+  constructor(private params: NavParams, private modalController: IModalController) {
+    this.config = this.params.get('config');
+  }
+
+  ngOnInit(): void {
+    this.title = this.config.title;
+    this.message = this.config.message;
+
+    if (this.config.icon) {
+      this.iconName = this.config.icon.name;
+      this.iconThemeColor = this.config.icon.themeColor;
+    }
+
+    if (this.config.okBtnText) {
+      console.warn('okBtnText will be deprecated on next major version. Please use okBtn instead.');
+      this.okBtn = this.config.okBtnText;
+    } else if (this.config.okBtn) {
+      if (typeof this.config.okBtn === 'string') {
+        this.okBtn = this.config.okBtn;
+      } else {
+        this.okBtn = this.config.okBtn.text;
+        this.okBtnIsDestructive = this.config.okBtn.isDestructive;

```

Comment:
This could be checked for in template instead;
`[isDestructive]="config?.okBtn?.isDestructive"`